sdk/spring/spring-cloud-azure-stream-binder-eventhubs/src/main/java/com/azure/spring/cloud/stream/binder/eventhubs/EventHubsMessageChannelBinder.java

```diff
@@ -122,7 +137,9 @@ protected MessageProducer createConsumerEndpoint(ConsumerDestination destination
                 destination.getName(), group, properties.getExtension().getCheckpoint());
         }
         inboundAdapter.setBeanFactory(getBeanFactory());
-
+        String instrumentationId = Instrumentation.buildId(CONSUMER, destination.getName() + "/" + (!INVALID_SUBSCRIPTION.equals(group) ? group : ""));
```

Comment:
for eventhubs, the consumer group is a required option, so we can ignore the case of a invalid group.